linux_docker_resources/Dockerfile-RHEL

```diff
@@ -1,9 +1,10 @@
-FROM almalinux:8
+FROM almalinux:9
 ARG ROS_DISTRO=rolling
+ARG EL_RELEASE=9
```

Comment:
I don't think this matters too much, but: don't we want to leave the default as 8 for now?  That is, we aren't quite ready to switch over to 9 yet.

Reply:
We're pretty darn close, but you're right. Done in 4c042d2.